build_projects/update-dependencies/UpdateFilesTargets.cs

```diff
@@ -87,7 +87,14 @@ public static BuildTargetResult ReplaceProjectJson(BuildTargetContext c)
         {
             List<DependencyInfo> dependencyInfos = c.GetDependencyInfos();
 
-            IEnumerable<string> projectJsonFiles = Directory.GetFiles(Dirs.RepoRoot, "project.json", SearchOption.AllDirectories);
+            IEnumerable<string> projectJsonFiles = new string[]
```

Comment:
While this feels like I hack I think I can live with it until we get core-setup repo changed.